code/script.js

```diff
@@ -5,5 +5,10 @@ function toggle() {
   this.classList.toggle("active")
 }
 
+
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
+
```

Comment:
Great, simple and clear! I see you have defined the meaning of 'toggle' as active first, and then selected 4 different elements from html to give the function and take away as they get clicked on.